netdeploy/networkTemplates_test.go

```diff
@@ -95,3 +97,22 @@ func TestValidate(t *testing.T) {
 	err = template.Validate()
 	a.NoError(err)
 }
+
+type overlayTestStruct struct {
+	A string
+	B string
+}
+
+// TestJsonOverlay ensures that encoding/json will only clobber fields present in the json and leave other fields unchanged
+func TestJsonOverlay(t *testing.T) {
```

Comment:
What is the purpose of this test? It looks like it is a json and dec test, and not a netdeploy test.
Ideally, the test should test `netdeploy/`.